Mathlib/MeasureTheory/Measure/MeasureSpace.lean

```diff
@@ -1010,6 +1010,59 @@ instance instCompleteLattice {_ : MeasurableSpace α} : CompleteLattice (Measure
 
 end sInf
 
+lemma inf_apply {s : Set α} (hs : MeasurableSet s) :
+    (μ ⊓ ν) s = sInf {m | ∃ t, m = μ (t ∩ s) + ν (tᶜ ∩ s)} := by
+  rw [← sInf_pair, Measure.sInf_apply hs, OuterMeasure.sInf_apply
```

Comment:
Could you put a few comments in the proof to explain what is going on? Otherwise, the argument isn't really easy to follow.